scripts/openstack/launch.py

```diff
@@ -0,0 +1,119 @@
+"""Launch script for openstack experiments.
+
+Requires a file that contains a list of IP addresses for instances that are:
+    - Turned on
+    - Accessible via ssh for the user of this file
+    - Configured with a predicators image
+    - Sufficient in number to run all of the experiments in the config file
+
+Usage example:
+    python scripts/openstack/launch.py --config example.yaml \
+        --machines machines.txt --sshkey ~/.ssh/cloud.key
```

Comment:
why can't machines.txt be autogenerated from the config?

Reply:
you can have persistent machines -- I have 30 static IP addresses right now on my account for example. the thought is that you would just make machines.txt once, or infrequently at least, and then run many different configs